pkgs/by-name/ch/charmcraft/package.nix

```diff
@@ -35,6 +35,7 @@ python3Packages.buildPythonApplication rec {
     humanize
     jinja2
     jsonschema
+    pip
```

Comment:
Is pip really required here?

Reply:
Yes, unfortunately so... Charmcraft does a bunch of python packaging stuff at runtime as part of its job, so this is needed.